api/src/main/resources/liquibase.xml

```diff
@@ -199,7 +199,7 @@
 		<comment>Move all Patients Ever enrolled in Facility Based Individual Management DSDM </comment>
 		<sql>
 			INSERT INTO patient_program (patient_id, program_id, date_enrolled, location_id, creator, date_created, voided, uuid)
-			select DISTINCT patient_id,5,encounter_datetime as date_enrolled,location_id,creator,date_created,0,uuid() as uuid from encounter where encounter_type=8;
+			select patient_id,5,encounter_datetime as date_enrolled,location_id,encounter.creator,encounter.date_created,0,uuid() as uuid from encounter INNER JOIN encounter_type ON(encounter_type.encounter_type_id=encounter.encounter_type) where encounter_type.uuid="8d5b27bc-c2cc-11de-8d13-0010c6dffd0f" group by patient_id;;
```

Comment:
@slubwama Don't hardcode the program id as 5 since there are places where other programs have already been used, use a join to program via the uuid 

Also for MySQL use single quotes for the uuid strings, capitalize SQL reserved words (SELECT, WHERE, GROUP BY)